Sources/GRInputField/UIKit/InputFieldView.swift

```diff
@@ -96,7 +96,7 @@ public class InputFieldView: UIView {
 
     private let textField = UITextField().then {
         $0.textAlignment = .left
-        $0.setContentHuggingPriority(.defaultLow, for: .horizontal)
```

Comment:
👀 

Priorita 249 nefunguje spravne? Tie hodnoty 0/1000 su take zradne niekedy (aj ked chapem co to ma v tomto pripade robit) 😄

Reply:
jj, mas pravdu, len som to vtedy uz tolko krat menil kym som dosiel ze co vlastne shrinkuje tie viewcka tak som to tu uz nechal :D zmenim